screens/IndividualMeals.js

```diff
@@ -0,0 +1,158 @@
+import { StyleSheet, Text, View, Pressable, Image, ImageBackground, ScrollView,SafeAreaView,StatusBar } from 'react-native'
+import {useState} from 'react'
+import colors from '../assets/colors/colors'
+import LoadingAnimation from './LoadingAnimation';
+import NutrientBar from './components/NutrientBar';
+
+
+export default IndividualMeals = ({navigation}) => {
+
+  const [calorie, setCalorie] = useState('lacking4');
+  const [sodium, setSodium] = useState('excessive6');
+  const [protein, setProtein] = useState('sufficient4');
+  const [fibre, setFibre] = useState('excessive12');
+  const [fat, setFat] = useState('lacking4');
+  const [carbohydrate, setCarbohydrate] = useState('sufficient5');
+  const [sugar, setSugar] = useState('lacking5');
+  const [cholesterol, setCholesterol] = useState('excessive5');
+  const [loading, setLoading] = useState(false);
+
+  const testing = new Date();
+  const dateTextWord = `${testing.getDate()}/${testing.getMonth()+1}/${testing.getFullYear()}`; 
+
+  if (loading) {
+    return <LoadingAnimation/>
+  }
+
+  return (
+
+    <View style={styles.container}>
+    <Image style={styles.foodphoto} source ={require("../assets/images/steak.png")}></Image>
+
+
+    <ImageBackground name = "Individual Icon" style = {styles.individual} source={require("../assets/images/dailyoverviewbg.png")}>
+      
+      <Text style = {styles.dailyOverviewText}>Individual Meals</Text>
```

Comment:
confusing style names. remember to rename them when copying and pasting in the future.